pkgs/development/php-packages/phalcon/default.nix

```diff
@@ -0,0 +1,27 @@
+{ buildPecl, lib, pcre2, fetchFromGitHub, php, pkg-config }:
+
+buildPecl rec {
+  pname = "phalcon";
+  version = "5.3.0";
+
+  src = fetchFromGitHub {
+    owner = "phalcon";
+    repo = "cphalcon";
+    rev = "v${version}";
+    sha256 = "sha256-82DJ+Qx0OYhw9Nv6FkAoyBec8WWfAiqNfBU9Ll/8RfA=";
+  };
+
+  internalDeps = [ php.extensions.session php.extensions.pdo ];
+
+  nativeBuildInputs = [ pkg-config ];
+  buildInputs = [ pcre2 ];
+
+  sourceRoot = "${src.name}/build/phalcon";
```

Comment:
Is this really necessary? Isn't the default when it's omitted?

Reply:
If it's omitted `sourceRoot` is set to `.` and phpize it's complaining it is not able to find `config.m4` as in this repo it's not located here https://github.com/phalcon/cphalcon/tree/master/build/phalcon

Reply:
> Is this really necessary? Isn't the default when it's omitted?

No, it isn't. This was explicitly challenged a short time ago.

Reply:
You are right, it's not needed. I don't know why it didn't work ealier without it. Removed

Reply:
If I remove the line then build fails with:

```
installing
post-installation fixup
checking for references to /private/tmp/nix-build-php-session-8.2.10.drv-0/ in /nix/store/iqciq9sabfxdwfrabcnz61cp8vw8l3km-php-session-8.2.10...
patching script interpreter paths in /nix/store/iqciq9sabfxdwfrabcnz61cp8vw8l3km-php-session-8.2.10
stripping (with command strip and flags -S) in  /nix/store/iqciq9sabfxdwfrabcnz61cp8vw8l3km-php-session-8.2.10/lib
checking for references to /private/tmp/nix-build-php-session-8.2.10.drv-0/ in /nix/store/1sr02hg7k3qvhpqxadm98xzv238nfli9-php-session-8.2.10-dev...
patching script interpreter paths in /nix/store/1sr02hg7k3qvhpqxadm98xzv238nfli9-php-session-8.2.10-dev
building '/nix/store/5xhpsnpb22qkpb31wcgqrv5rnq83qsb5-php-phalcon-5.3.0.drv'...
unpacking sources
unpacking source archive /nix/store/ihbmfqp0mzlharzfqq80zyn6w8wxaczd-source
source root is source
patching sources
autoreconfPhase
Cannot find config.m4. 
Make sure that you run '/nix/store/x3m43wlll7fqyi84gdzsfd08i3rrxpcn-php-8.2.10-dev/bin/phpize' in the top level source directory of the module

error: builder for '/nix/store/5xhpsnpb22qkpb31wcgqrv5rnq83qsb5-php-phalcon-5.3.0.drv' failed with exit code 1;
       last 8 log lines:
       > unpacking sources
       > unpacking source archive /nix/store/ihbmfqp0mzlharzfqq80zyn6w8wxaczd-source
       > source root is source
       > patching sources
       > autoreconfPhase
       > Cannot find config.m4.
       > Make sure that you run '/nix/store/x3m43wlll7fqyi84gdzsfd08i3rrxpcn-php-8.2.10-dev/bin/phpize' in the top level source directory of the module
       >
       For full logs, run 'nix-store -l /nix/store/5xhpsnpb22qkpb31wcgqrv5rnq83qsb5-php-phalcon-5.3.0.drv'.
```